### PR TITLE
Accept style attr and Qwik class array

### LINKED ISSUE #151: Types don't work with Qwik's built-in class array syntax
image

With a Qwik component, you can do this:

```tsx
<Form class={[classOne, classTwo]} />
```
Modular Forms types `class` as `Maybe<string>`. Can we make this `Maybe[string]|Maybe[string[]]`?

### PR DESCRIPTION
**Pull request by brandonpittman**
Fixes #151

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you! I will try to merge this and publish a new version today or tomorrow.

File: packages/qwik/src/components/Form.tsx [hunk 1]
```diff
@@ -49,7 +49,8 @@ export type FormProps<
 
   // HTML props
   id?: Maybe<string>;
-  class?: Maybe<string>;
+  class?: Maybe<string | string[]>;
```
[the remark this answers is not in the document]
Comment (brandonpittman) [reply packages/qwik/src/components/Form.tsx]: Changed it in [e459071](https://github.com/fabian-hiller/modular-forms/pull/152/commits/e459071dbd7f261bac6110de017047d763527e32)

**Comment by fabian-hiller** [comment]
Looks good now. Thank you! I plan to merge it later and publish a new version with the changes.

**Comment by fabian-hiller** [comment]
v0.21.1 is available

**Pull request merged**